22_23_10/script.js

```diff
@@ -0,0 +1,61 @@
+const form = document.getElementById("mainForm");
+
+class InputParent {
+  constructor(name, type) {
```

Comment:
Лучше передавать параметры объектом. 
```suggestion
  constructor({name, type}) {
```